src/test/java/seedu/address/logic/parser/expenditure/ExpExpEditCommandParserTest.java

```diff
@@ -117,10 +117,10 @@ public void parse_allFieldsSpecified_success() {
         String userInput = targetIndex.getOneBased() + ID_DESC_BOB + TAG_DESC_HUSBAND
                 + AMOUNT_DESC_AMY + DATE_DESC_AMY + INFO_DESC_AMY + TAG_DESC_FRIEND;
 
-        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder().withInfo(VALID_INFO_AMY)
+        ExpEditCommand.EditExpenditureDescriptor descriptor = new EditPersonDescriptorBuilder().withInfo(VALID_INFO_AMY)
```

Comment:
i think you can just use `EditExpenditureDescriptor`, since you have imported it already